programs/libraries/rjson.scl

```diff
@@ -0,0 +1,109 @@
+//raw-json string to text
+if(0,    //if you are using an old version, change it to 1
+rjson(rj)->(
+barname='chyxrjson:rjson'+(rand(0.9)-'.');

```

Comment:
Could you please use indentation, cos it makes the code easier to read

Reply:
ok.